src/list_ordered_multimap.rs

```diff
@@ -3014,6 +3031,70 @@ impl<'map, Key, Value> Iterator for IterMut<'map, Key, Value> {
     }
 }
 
+/// An iterator that owns and yields all key-value pairs in a multimap. The order of
```

Comment:
Can you add a note that this will clone keys and may be potentially expensive.

Reply:
Yes.